exporter/awsemfexporter/metric_translator.go

```diff
@@ -61,6 +62,14 @@ type CWMetrics struct {
 	Fields       map[string]interface{}
 }
 
+type GroupedCWMetric struct {
+	Namespace    string
+	Metrics      map[string]interface{}
+	Timestamp    int64
+	Dimensions   map[string]interface{}
+	MetricUnits  map[string]string
+}
```

Comment:
From the design doc, should `GroupedMetric` only have `Timestamp`, `Fields`, and `Metrics` as attributes?